containerbuddy/config.json

```diff
@@ -1,11 +1,12 @@
 {
-  "consul": "consul:8500",
+  "consul": "{{.CONSUL}}:8500",
+  "onStart": "/opt/containerbuddy/onstart.sh",
   "services": [
     {
-      "name": "wordpress",
+      "name": "app",
```

Comment:
You can pass the configuration as a json string: https://github.com/joyent/containerbuddy#configuring-containerbuddy